tests/test_tofu.py

```diff
@@ -13,7 +20,7 @@ def _check_dropped_qs(granularity, stratify, forget_random, debug_dict):
     """check correct number of questions dropped per author/indices lie in range of
     forget authors -> not needed if random"""
 
-    forgot_q_frac = 0.1
+    forgot_q_frac = 0.34
```

Comment:
Is this necessary if defined globally? 
```suggestion
```

Reply:
ah yes I missed removing that one, thanks!